src/cobra/core/reaction.py

```diff
@@ -914,7 +1030,12 @@ def add_metabolites(self, metabolites_to_add, combine=True, reversibly=True):
                     )
                 )
 
-    def subtract_metabolites(self, metabolites, combine=True, reversibly=True):
+    def subtract_metabolites(
+        self,
+        metabolites: Dict[Metabolite, Union[int, float]],
+        combine: Optional[bool] = True,
```

Comment:
Please make the type as `bool` and document the optional behaviour in the docstring.